packages/db/src/schema/job.ts

```diff
@@ -23,6 +23,7 @@ export const jobStatus = pgEnum("job_status", [
   "invalid_job_agent",
   "invalid_integration",
   "external_run_not_found",
+  "triggered",
```

Comment:
why did we do this? we did not talk about adding a new state. In my messages I said we would use pending to indicated this, and switch it to in_progress